scripts/variantstore/wdl/GvsExtractCohortFromSampleNames.wdl

```diff
@@ -99,9 +93,22 @@ workflow GvsExtractCohortFromSampleNames {
   }
 
   File cohort_sample_names_file = select_first([write_array_task.output_file, cohort_sample_names])
+  Int effective_sample_count = length(read_lines(cohort_sample_names_file))
+
+  Int effective_scatter_count = if defined(extract_scatter_count_override) then select_first([extract_scatter_count_override])
+                                else if is_wgs then
+                                     if effective_sample_count < 5000 then 1 # This results in 1 VCF per chromosome.
+                                     else if effective_sample_count < 20000 then 2000 # Stroke Anderson
```

Comment:
I think we can retire the Stroke Anderson comment since the data has long been taken

Reply:
😭